discovery-infra/assisted_service_api.py

```diff
@@ -14,9 +18,51 @@ def __init__(self, inventory_url):
         self.inventory_url = inventory_url
         configs = Configuration()
         configs.host = self.inventory_url + "/api/assisted-install/v1"
+        if 'OFFLINE_TOKEN' in os.environ:
```

Comment:
not sure why we need "if" here. looks like if there is no token function will just return